python/mxnet/gluon/utils.py

```diff
@@ -145,12 +143,12 @@ def group_by_ctx(arr_list):
     all_ctx_sum = []
     ctx = arrays[0].context
     for group in arrays_groups:
-        sum_sq = mx.nd.multi_sum_sq(*arrays_groups[group],
-                                    num_arrays=len(arrays_groups[group]))
-        sum_sq = nd.add_n(*sum_sq)
+        sum_sq = ndarray.multi_sum_sq(*arrays_groups[group],
```

Comment:
What’s the difference between mxnet.ndarray and ndarray here?

Reply:
I think this is equivalent,
so I modified it to ndarray, since this was already imported for other functions in the file